src/Screens/CartScreen/CartScreen.jsx

```diff
@@ -1,14 +1,15 @@
-import { useState, useEffect } from "react";
-import { useData } from "../../Helper";
 import "./CartScreen.css";
-import { CardHorizontal } from "../../Components";
-import { CheckOutDetails } from "../../Components";
+import { useState, useEffect } from "react";
+import { CardHorizontal, CheckOutDetails } from "../../Components";
 import { EmptyCart } from "../";
+import { useAuth } from "../../Helper";
 
 const CartScreen = () => {
   const {
-    state: { cart },
-  } = useData();
+    userState: {
+      userData: { cart },
+    },
+  } = useAuth();
 
   const [totalPrice, setTotalPrice] = useState(0);
```

Comment:
would like to talk about the use of these two states here. maybe you don't need them.

Reply:
fixed in https://github.com/saadahmadhere/elmart/pull/13/commits/d02b1a8531f501320f11b2d2f643a74250f96187